CHANGELOG.asciidoc

```diff
@@ -74,6 +74,7 @@ This release also includes changes from <<release-3-2-8, 3.2.8>>.
 * TINKERPOP-1357 Centrality Recipes should mention pageRank and OLAP.
 * TINKERPOP-1489 Provide a Javascript Gremlin Language Variant
 * TINKERPOP-1586 SubgraphStrategy in OLAP
+* TINKERPOP-1628 Implement TraversalSelectStep.
```

Comment:
i know i pretty much always have false alarms about your changelog entries, but this one looks amiss. we generate this portion of the changelog on release so they shouldn't be edited. you just need an entry on 3.3.3 like we normally have.